cnf-tests/testsuites/pkg/pods/pods.go

```diff
@@ -103,6 +103,26 @@ func RedefineWithLabel(pod *corev1.Pod, key, value string) *corev1.Pod {
 	return pod
 }
 
+// RedefineWithRestrictedPrivileges enforces restricted privileges on the pod.
+func RedefineWithRestrictedPrivileges(pod *corev1.Pod) *corev1.Pod {
+	pod.Spec.SecurityContext = &corev1.PodSecurityContext{
+		SeccompProfile: &corev1.SeccompProfile{Type: corev1.SeccompProfileTypeRuntimeDefault},
+		FSGroup:        pointer.Int64Ptr(801),
```

Comment:
Please, explain why 801 is a good number when https://github.com/openshift-kni/cnf-features-deploy/pull/1591 is merged:

```suggestion
		// 801 is the groupID which dpdk-user belongs to. See `tools/s2i-dpdk/Dockerfile`
		FSGroup:        pointer.Int64Ptr(801),
```

Reply:
no, if we merge 1591. This is not needed. I will delete it

Reply:
done